.github/workflows/test_tutor_oars.yml

```diff
@@ -0,0 +1,34 @@
+name: Test Tutor OARS
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  TUTOR_ROOT: ./.ci/
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - name: Install python reqs
+        run: pip install -r requirements/dev.txt
+      - name: Install pars
+        run: pip install -e .
+      - name: Tutor local launch
+        run: tutor config save
+      - name: Tutor build openedx
+        run: tutor images build openedx
```

Comment:
This can be painful, so let's make it require CC approval and configure a local system cache for it.

Also, remember that for dev is `tutor images build openedx-dev`

Reply:
Yeah this is pretty experimental, just trying to get it to a working state. I hear there are new tutor capabilities that may help this as well, so I'm looking into that next.

Reply:
Yep, also here are some CI processes we can use:

https://github.com/overhangio/test-docker-build/blob/main/.github/workflows/build.yml